app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -125,16 +155,29 @@ class HomeFragment : Fragment() {
         @JvmStatic
         fun create() = HomeFragment()
     }
+
+    fun getFocusedTilePosition(): Int {
+        return (activity.currentFocus.parent as? RecyclerView)?.getChildAdapterPosition(activity.currentFocus) ?: RecyclerView.NO_POSITION
+    }
+
+    fun dispatchKeyEvent(event: KeyEvent): Boolean {
+        if (event.keyCode == KeyEvent.KEYCODE_DEL && getFocusedTilePosition() != RecyclerView.NO_POSITION) {
+            activity.openContextMenu(view)
+            return true
+        }
+        return false
+    }
 }
 
 private class HomeTileAdapter(
         private val uiLifecycleCancelJob: Job,
-        private val tiles: List<HomeTile>,
+        tiles: MutableList<HomeTile>,
         private val onUrlEnteredListener: OnUrlEnteredListener
 ) : RecyclerView.Adapter<TileViewHolder>() {
+    var homeTiles = tiles
```

Comment:
No need to reassign this, I think - can you just keep the change from above (`private val tiles`) and either rename to `homeTiles` if you want it to be named that way, and change it to `var` if you want to be able to mutate it?

By putting `private val` in the arugment list, you can automatically assign it to a variable and just use that.